services/QuillLMS/app/workers/report_partner_session_completion_worker.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ReportPartnerSessionCompletionWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: SidekiqQueue::DEFAULT
+
+  def perform(activity_session_id)
```

Comment:
I'd say use the PartnerSession id as the param, but even without that change, this could be
```ruby
ActivitySessionPartnerMetadata.find_by(activity_session_id:)&.send_to_partner